packages/docs/pages/reference/elements/table.mdx

```diff
@@ -8,12 +8,170 @@ import {
 
 # `<table>`
 
-Short description of what the `<table>` tag does.
+The `<table>` tag produces a container for the [`<tabular>`](./tabular) tag and assigns to it a number which can be associated with a title for table. The [`<tabular>`](./tabular) tag produces what one normally thinks of as a table
+consisting of rows and columns.  For complex tables, there is an [online table
+builder](https://www.latex-tables.com/) which can generate PreTeXt output.
 
 ## Syntax
 
 <SyntaxDisplay name="table" />
 
 ## Examples
 
-    
\ No newline at end of file
+### A comment about table borders
+
+Placing borders in tables exactly where you want them requires that you
+understand in which elements the attributes `top`, `left`, `bottom`, and
+`right` may be applied. Note that while horizontal borders (rules)
+ are easy, vertical rules have been made intentionally more work "to gently
+ guide authors towards good choices" in document design.
+[Table 4.18.3]
```

Comment:
Change to `The PreTeXt Guide's [section on tables](...) is the definitive source`. It looks like the link is autogenerated. Maybe there is a link to that section that will be permanent via a `label` attribute?